composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "gquemener/prooph-pack",
+    "description": "A pack to install Prooph PDO event sourcing components in a Symfony Flex application",
+    "type": "symfony-pack",
+    "license": "MIT",
```

Comment:
Should it be BSD licensed (like [the event store bundle](https://github.com/prooph/event-store-symfony-bundle/blob/master/LICENSE.md))?